nautobot/extras/templates/extras/jobresult.html

```diff
@@ -46,7 +46,9 @@
             </a>
         {% endif %}
     {% endif %}
-    {% export_button %}
+    <a href="{% url 'extras-api:joblogentry-list' %}?job_result={{ result.pk }}&format=csv" class="btn btn-success">
+        <span class="mdi mdi-database-export" aria-hidden="true"></span> Export
+    </a>
```

Comment:
The `export` button on the JobResult detail view doesn't export the JobResult itself; it exports the JobLogEntries associated with this result. Previously (see a few files below) this was done via a custom `export` handler on the JobResultView; now it's cleaner. :-)